lib/generators/heroku_installer.rb

```diff
@@ -124,7 +124,10 @@ def add_sidekiq_config
       def add_readme_info
         append_to_file("README.md", <<-README_INFO)
 ### How to deploy
-1. Use the "Deploy to Heroku" button you'll find in this file
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+1. Use the "Deploy to Heroku" button
```

Comment:
No, Markdown does not care about this and auto-converts it to `<ol>`, so you can keep using 1, 1, 1, 1 and never need to modify the steps manually if you need to add some step in the middle.